component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1027,14 +1027,14 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 						}
 						if (stripos($fieldval, "%d") !== false)
 						{
-							$days = intval($timedelta / (60 * 60 * 24));
+							$days = (int) $timedelta / (60 * 60 * 24);
```

Comment:
I've not seen any performance comparisons online that are less than 3-4 years old.

I'll pass it for now but would be interested in seeing the basis of your claim ;)